src/main/java/seedu/loyaltylift/logic/commands/EditCustomerCommand.java

```diff
@@ -19,11 +19,11 @@
 import seedu.loyaltylift.commons.util.CollectionUtil;
 import seedu.loyaltylift.logic.commands.exceptions.CommandException;
 import seedu.loyaltylift.model.Model;
-import seedu.loyaltylift.model.customer.Address;
+import seedu.loyaltylift.model.attribute.Address;
```

Comment:
The import order needs to reorganised. There should be 11 files with this issue.